vite.config.ts

```diff
@@ -61,6 +61,9 @@ export default ({ mode, command }: ConfigEnv): UserConfigExport => {
         targets: fileCopyTargets,
       }),
     ],
+    define: {
+      APP_VERSION: JSON.stringify(process.env.npm_package_version),
```

Comment:
Is not it a simple string?

Reply:
I tried to remove `JSON.stringif`y from the `process.env.npm_package_version` since it should be a string, but for some reason, it crashes the whole app
It says `Uncaught SyntaxError: Unexpected number`
So it might retrieve it as a number after all :)

Reply:
https://vitejs.dev/config/shared-options.html#define